### DISCUSSION TITLE
Welcome to DIDP Discussions!

### DISCUSSION THREAD

**Discussion by anubhav-cs** [domain-independent-dp/didp-rs#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other community members. We hope that you use GitHub discussion to:
  * ask any questions about domain-independent DP,
  * share ideas,
  * deliberate over improvements to DIDP.

## Guidelines

  * If you have any questions about performance issues, please share a program encoding your smallest problem.
  * Please refer to the DIDP tutorial in this repository for help on modeling problems in DIDP.

We want to keep the DIDP issues, which we use to track bugs and accepted change requests, separate from DIDP discussions. So, we will convert an issue into a discussion if it requests a change requiring further deliberation or does not involve code modification.

Cheers,
DIDP Team

**Comment by matheusdiogenesandrade** [comment]
If you could elaborate more on the differences between DIDP, Decision Diagrams, and Markov Decision Processes, what would you point out?

**Reply by Kurorororo** [reply]
Chapter 2 of my Ph.D. thesis may help you: https://tidel.mie.utoronto.ca/pubs/Theses/Kuroiwa_Ryo_20249_PhD_thesis.pdf
